solidity/contracts/integrator/AbstractTBTCDepositor.sol

```diff
@@ -134,10 +136,10 @@ abstract contract AbstractTBTCDepositor {
         IBridgeTypes.BitcoinTxInfo calldata fundingTx,
         IBridgeTypes.DepositRevealInfo calldata reveal,
         bytes32 extraData
-    ) internal returns (uint256) {
+    ) internal returns (uint256 depositKey, uint256 initialDepositAmount) {
```

Comment:
Let's make sure we are covering this change in the example described in the contract's docstring.

Reply:
a3a7cf6b97139fc3af4e57523c7ccade68354d69